plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryConnectorTest.java

```diff
@@ -240,11 +247,11 @@ public void testEmptyProjection(TableDefinition.Type tableType, String createSql
     public Object[][] emptyProjectionSetupDataProvider()
     {
         Object[][] testCases = new Object[][] {
-                {TABLE, "CREATE TABLE %s AS SELECT * FROM tpch.region", "DROP TABLE %s"},
-                {VIEW, "CREATE VIEW %s AS SELECT * FROM tpch.region", "DROP VIEW %s"},
-                {MATERIALIZED_VIEW, "CREATE MATERIALIZED VIEW %s AS SELECT * FROM tpch.region", "DROP MATERIALIZED VIEW %s"},
-                {EXTERNAL, "CREATE EXTERNAL TABLE %s OPTIONS (format = 'CSV', uris = ['gs://" + gcpStorageBucket + "/tpch/tiny/region.csv'])", "DROP EXTERNAL TABLE %s"},
-                {SNAPSHOT, "CREATE SNAPSHOT TABLE %s CLONE tpch.region", "DROP SNAPSHOT TABLE %s"},
+                {TABLE, "CREATE TABLE %s %s", "AS SELECT * FROM tpch.region", "DROP TABLE %s"},
+                {VIEW, "CREATE VIEW %s %s", "SELECT * FROM tpch.region", "DROP VIEW %s"},
+                {MATERIALIZED_VIEW, "CREATE MATERIALIZED VIEW %s %s", "AS SELECT * FROM tpch.region", "DROP MATERIALIZED VIEW %s"},
+                {EXTERNAL, "CREATE EXTERNAL TABLE %s %s", "OPTIONS (format = 'CSV', uris = ['gs://" + gcpStorageBucket + "/tpch/tiny/region.csv'])", "DROP EXTERNAL TABLE %s"},
+                {SNAPSHOT, "CREATE SNAPSHOT TABLE %s %s", "CLONE %s", "DROP SNAPSHOT TABLE %s"},
```

Comment:
couldn't think of a cleaner alternative than parameterizing the "table definition" as well. Suggestions welcome.

Reply:
Fair point.